test_alice_locks_tokens fails, locally & remotely

### Describe the bug

Run the following, and you'll see it fail.
```console
brownie test util/test/veOcean/test_lock.py::test_alice_locks_tokens -s
```

Tracebacks below.

It also fails remotely. [Example log](https://github.com/oceanprotocol/df-py/actions/runs/4944277372/jobs/8839584379?pr=567)

### Traceback - Local

```text
(venv) trentmc@trentmc-xps:~/code/df-py$ brownie test util/test/veOcean/test_lock.py::test_alice_locks_tokens -s
Brownie v1.19.3 - Python development framework for Ethereum

======================================================================================== test session starts ========================================================================================
platform linux -- Python 3.10.6, pytest-6.2.5, py-1.11.0, pluggy-1.0.0 -- /home/trentmc/code/df-py/venv/bin/python3
cachedir: .pytest_cache
hypothesis profile 'brownie-verbose' -> verbosity=2, deadline=None, max_examples=50, stateful_step_count=10, report_multiple_bugs=False, database=DirectoryBasedExampleDatabase(PosixPath('/home/trentmc/.brownie/hypothesis'))
rootdir: /home/trentmc/code/df-py, configfile: pytest.ini
plugins: eth-brownie-1.19.3, web3-5.31.3, hypothesis-6.27.3, forked-1.4.0, xdist-1.34.0
collected 1 item                                                                                                                                                                                    

util/test/veOcean/test_lock.py::test_alice_locks_tokens RUNNING
mnemonic: 'music galaxy jungle walk author person uncover tired bean acid ripple feel'
mnemonic: 'radio battle hour spot pave case virtual protect chaos brain bid evil'
ERROR

============================================================================================== ERRORS ===============================================================================================
_____________________________________________________________________________ ERROR at setup of test_alice_locks_tokens _____________________________________________________________________________

    @enforce_types
    def setup_function():
        networkutil.connect(networkutil.DEV_CHAINID)
        oceanutil.recordDevDeployedContracts()
        global accounts, alice, bob, veOCEAN, OCEAN
        accounts = brownie.network.accounts
    
        alice = accounts.add()
        bob = accounts.add()
    
        OCEAN = oceanutil.OCEANtoken()
>       veOCEAN = B.veOcean.deploy(
            OCEAN.address, "veOCEAN", "veOCEAN", "0.1.0", {"from": alice}
        )
E       ValueError: insufficient funds for gas * price + value

util/test/veOcean/test_lock.py:67: ValueError
====================================================================================== short test summary info ======================================================================================
ERROR util/test/veOcean/test_lock.py::test_alice_locks_tokens - ValueError: insufficient funds for gas * price + value
========================================================================================= 1 error in 2.17s =======================================
```

### Traceback - Remote

```text
___________________________ test_alice_locks_tokens ____________________________

    @enforce_types
    def test_alice_locks_tokens():
        """Lock tokens then check balance."""
        veOCEAN.checkpoint()
        OCEAN.approve(veOCEAN.address, TA, {"from": alice})
    
        t0 = chain.time()
        t1 = t0 // WEEK * WEEK + WEEK
        t2 = t1 + WEEK
        chain.sleep(t1 - t0)
    
        assert OCEAN.balanceOf(alice) != 0
    
        veOCEAN.create_lock(TA, t2, {"from": alice})
    
        assert OCEAN.balanceOf(alice) == 0
    
        epoch = veOCEAN.user_point_epoch(alice)
        assert epoch != 0
    
        assert veOCEAN.get_last_user_slope(alice) != 0
>       aliceVotingPower = (veOCEAN.balanceOf(alice, chain.time())) / toBase18(1.0)
E       brownie.exceptions.VirtualMachineError: revert

util/test/veOcean/test_lock.py:42: VirtualMachineError
```

--- a/util/test/veOcean/test_lock.py
+++ b/util/test/veOcean/test_lock.py
@@ -12,8 +12,10 @@ alice = None
 bob = None
 veOCEAN = None
 OCEAN = None
-WEEK = 7 * 86400
-MAXTIME = 4 * 365 * 86400  # 4 years
+DAY = 86400
+WEEK = 7 * DAY
+YEAR = 365 * DAY
+MAXTIME = 4 * YEAR
 chain = brownie.network.chain
 TA = toBase18(10.0)
 
@@ -26,8 +28,9 @@ def test_alice_locks_tokens():
 
     t0 = chain.time()
     t1 = t0 // WEEK * WEEK + WEEK
-    t2 = t1 + WEEK
+    t2 = t1 + YEAR
     chain.sleep(t1 - t0)
+    chain.mine()
 
     assert OCEAN.balanceOf(alice) != 0
 
@@ -40,10 +43,10 @@ def test_alice_locks_tokens():
 
     assert veOCEAN.get_last_user_slope(alice) != 0
     aliceVotingPower = (veOCEAN.balanceOf(alice, chain.time())) / toBase18(1.0)
-    expectedVotingPower = (TA * WEEK / MAXTIME) / toBase18(1.0)
+    expectedVotingPower = (TA * YEAR / MAXTIME) / toBase18(1.0)
     assert aliceVotingPower == approx(expectedVotingPower, 0.5)
 
-    brownie.network.chain.sleep(t2)
+    chain.sleep(t2 - t1)
     chain.mine()
 
     veOCEAN.withdraw({"from": alice})

--- a/util/test/veOcean/test_lock.py
+++ b/util/test/veOcean/test_lock.py
@@ -21,7 +21,7 @@ TA = toBase18(10.0)
 @enforce_types
 def test_alice_locks_tokens():
     """Lock tokens then check balance."""
-    veOCEAN.checkpoint()
+    veOCEAN.checkpoint({"from": alice})
     OCEAN.approve(veOCEAN.address, TA, {"from": alice})
 
     t0 = chain.time()
@@ -60,8 +60,8 @@ def setup_function():
     global accounts, alice, bob, veOCEAN, OCEAN
     accounts = brownie.network.accounts
 
-    alice = accounts.add()
-    bob = accounts.add()
+    alice = accounts[0]
+    bob = accounts[1]
 
     OCEAN = oceanutil.OCEANtoken()
     veOCEAN = B.veOcean.deploy(

--- a/util/test/veOcean/test_lock.py
+++ b/util/test/veOcean/test_lock.py
@@ -60,8 +60,11 @@ def setup_function():
     global accounts, alice, bob, veOCEAN, OCEAN
     accounts = brownie.network.accounts
 
-    alice = accounts[0]
-    bob = accounts[1]
+    alice = accounts.add()
+    bob = accounts.add()
+
+    accounts[0].transfer(alice, "0.01 ether")
+    accounts[0].transfer(bob, "0.01 ether")
 
     OCEAN = oceanutil.OCEANtoken()
     veOCEAN = B.veOcean.deploy(

